docs/guides/exporters/oci-docker.md

```diff
@@ -0,0 +1,54 @@
+# OCI and Docker exporters
+
+The `oci` exporter outputs the build result into an
+[OCI image layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md)
+tarball. The `docker` exporter behaves the same way, except it exports a Docker
+image layout instead.
+
+The [`docker` driver](../drivers/docker.md) doesn't support these exporters. You
+must use `docker-container` or some other driver if you want to generate these
+outputs.
+
+## Synopsis
+
+Build a container image using the `oci` and `docker` exporters:
+
+```console
+$ docker buildx build --output type=oci[,parameters] .
+$ docker buildx build --output type=docker[,parameters] .
+```
+
+The following table describes the available parameters:
+
+| Parameter           | Value                                  | Default | Description                                                                                        |
+| ------------------- | -------------------------------------- | ------- | -------------------------------------------------------------------------------------------------- |
+| `name`              | String                                 |         | Specify image name(s)                                                                              |
+| `dest`              | String                                 |         | Path                                                                                               |
+| `compression`       | `uncompressed`,`gzip`,`estargz`,`zstd` | `gzip`  | Compression type, see [compression][1]                                                             |
+| `compression-level` | `0..22`                                |         | Compression level, see [compression][1]                                                            |
+| `force-compression` | `true`,`false`                         | `false` | Forcefully apply compression, see [compression][1]                                                 |
+| `oci-mediatypes`    | `true`,`false`                         | `false` | Use OCI mediatypes in exporter manifests, see [OCI Media types][2]                                 |
```

Comment:
Woops, this is my bad.

`oci-mediatypes` default value depends on `oci`/`docker`. It's `true` for `oci` and `false` for `docker`.